backend/core/views.py

```
@@ -4707,23 +4708,41 @@ def export(self, request, pk):
-                                    os.path.basename(evidence.attachment.name),
-                                ),
-                                default_storage.open(evidence.attachment.name).read(),
-                            )
-                zipf.writestr("index.html", index_content)
 
-            response = FileResponse(open(zip_name, "rb"), as_attachment=True)
-            response["Content-Disposition"] = f'attachment; filename="{zip_name}"'
-            os.remove(zip_name)
-            return response
+            # Create temporary file that will be automatically deleted
+            temp_file = tempfile.NamedTemporaryFile(delete=True, suffix=".zip")
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)